src/Package/MSBuild.UnGAC.Setup/MSBuild.UnGAC.Setup.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- The project MUST be net45. -->
+    <TargetFramework>net45</TargetFramework>
+    <!-- Set as an exe because this project publishes its own output. -->
+    <OutputType>Exe</OutputType>
+    <!-- Forcing the 'vsix' output to be json output. -->
+    <TargetVsixContainerName>MSBuild.UnGAC.Setup.json</TargetVsixContainerName>
+    <!-- VS Insertion -->
+    <VisualStudioInsertionComponent>Microsoft.Build.UnGAC.Setup</VisualStudioInsertionComponent>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_SwixArgs Include="PackageType=manifest"/>
+    <!-- If we don't set TargetExt to .json, arcade defaults to generating a .vsix -->
+    <_SwixArgs Include="TargetExt=.json"/>
+  </ItemGroup>
+
+  <!-- SwrProperty is an arcade concept here. -->
+  <ItemGroup>
+    <!-- BinDir is set via SwrProperty so arcade can see it. OutDir is the path to the exe. See exe.swr to see how this is used. -->
```

Comment:
```suggestion
    <!-- BinDir is set via SwrProperty to be passed to the swr compiler. OutDir is the path to the bin folder. See exe.swr to see how this is used. -->
```

Personally I'd cut out the last two sentences but no real harm in leaving them in.